### ISSUE TITLE
protoc-3.5.0-linux-x86_64 introduced GLIBC_2.14 dependency

### ISSUE THREAD

**Issue by zhangkun83** [google/protobuf#4138]
```
$ objdump -p protoc-3.4.0-linux-x86_64.exe | grep LIBC
    0x0d696913 0x00 07 GLIBC_2.3
    0x0d696914 0x00 06 GLIBC_2.4
    0x0d696913 0x00 04 GLIBC_2.3
    0x09691a75 0x00 03 GLIBC_2.2.5
    0x09691a75 0x00 05 GLIBC_2.2.5
    0x09691972 0x00 02 GLIBC_2.3.2

$ objdump -p protoc-3.5.0-linux-x86_64.exe | grep LIBC
    0x0d696913 0x00 10 GLIBC_2.3
    0x09691a75 0x00 06 GLIBC_2.2.5
    0x0d696914 0x00 09 GLIBC_2.4
    0x06969194 0x00 08 GLIBC_2.14
    0x09691974 0x00 07 GLIBC_2.3.4
    0x0d696913 0x00 04 GLIBC_2.3
    0x09691a75 0x00 03 GLIBC_2.2.5
    0x09691a75 0x00 05 GLIBC_2.2.5
    0x09691972 0x00 02 GLIBC_2.3.2
```
32-bit is not affected.

Because of this, the released 64-bit 3.5.0 protoc artifact can't be run in the gRPC Java release docker container, which runs CentOS 6.6 shipped with GLIBC_2.12. Is this an intentional change?

**Comment by zhangkun83** [comment]
@pherl

**Comment by pherl** [comment]
Should have been fixed in 3.5.1-1
